examples/apps/contact-collection/package.json

```diff
@@ -12,12 +12,11 @@
   "license": "MIT",
   "author": "Microsoft and contributors",
   "main": "dist/index.js",
-  "module": "lib/index.js",
+  "module": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint",
-    "build:compile": "concurrently npm:tsc npm:build:esnext",
-    "build:esnext": "tsc --project ./tsconfig.esnext.json",
+    "build:compile": "npm run tsc",
```

Comment:
Since the target is actually for `esnext`, should we retain the `esnext` build instead of the `tsc` build instead.

Reply:
I have changed the name of the npm script accordingly. I think this is what you wanted.

I did not restore the esnext specific tsconfig file name: tooling (like vscode ) likes there to be one with the default name, and I figured if we have only one, it might as well use the default name.